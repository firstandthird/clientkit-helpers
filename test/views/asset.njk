{{ 'test.jpg' | asset }}
{{ 'test.jpg' | asset(2) }}
