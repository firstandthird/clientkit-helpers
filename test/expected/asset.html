/test.jpg
/test.jpg?v=2
